shared/chat/conversation/messages/wrapper/unfurl/generic/index.js

```diff
@@ -55,23 +62,32 @@ class UnfurlGeneric extends React.Component<Props> {
           {!!this.props.description && (
             <Kb.Text type="Body" lineClamp={5}>
               {this.props.description}
+              {!Styles.isMobile && showBottomImage && (
+                <React.Fragment>
+                  {' '}
+                  <Kb.Icon
+                    boxStyle={styles.collapseBox}
+                    onClick={this.props.onCollapse}
+                    type={this.props.isCollapsed ? 'iconfont-caret-right' : 'iconfont-caret-down'}
+                  />
+                </React.Fragment>
+              )}
             </Kb.Text>
           )}
-          {!!this.props.imageURL &&
-            !!this.props.imageHeight &&
-            !!this.props.imageWidth &&
-            !this.props.showImageOnSide && (
+          {showBottomImage && !this.props.isCollapsed && (
+            <Kb.Box2 direction="vertical" fullWidth={true}>
               <UnfurlImage
-                url={this.props.imageURL}
+                url={this.props.imageURL || ''}
```

Comment:
comment around here about `showBottomImage` asserting these are set would be nice

Reply:
I was annoyed Flow didn't figure this out.

Reply:
flow is correct here i believe. just cause you check doesnt' mean something can't come in and just change it aka `this.props.imageUrl = null`. if you really want to just make a const like
`const goodImageURL`, then checks against that should be resolved fine (since its const)